core/os/os.cpp

```diff
@@ -707,9 +707,9 @@ const char *OS::get_video_driver_name(int p_driver) const {
 	switch (p_driver) {
 		case VIDEO_DRIVER_GLES2:
 			return "GLES2";
-		case VIDEO_DRIVER_GLES3:
```

Comment:
So just for clarification, does this mean GLES3 is being removed before Vulkan is finished? This is important to know for anyone who builds from master between 3.2 and 4.0.

Reply:
Yes, once the `vulkan` branch is merged in `master` after the `3.2-stable` release, GLES3 will be no more. It will live on in the `3.2` branch of course.